src/js/observations.js

```diff
@@ -0,0 +1,12 @@
+/*jshint esversion: 6 */
+class Observations {  
```

Comment:
If this is something that is to be embedded in an MPage in the future, I ran into issues with using the ES6 class within Powercharts IE 10.  You'd think the transpiling of the code would handle it but I would validate that you can or cannot use this format within Powerchart.

https://babeljs.io/docs/usage/caveats/